test/directoryAsModule.js

```diff
@@ -0,0 +1,8 @@
+var assert = require('assert');
+var requireDir = require('..');
+
+assert.deepEqual(requireDir('./directoryAsModule'), {
+    a: 'a', b: 'b'
+});
+
+console.log('Automatic Index tests passed')
```

Comment:
Minor: let's have this `console.log` message be consistent with the feature & test name: "Directory as module tests passed".